src/main/java/bio/terra/service/common/azure/StorageTableName.java

```diff
@@ -0,0 +1,59 @@
+package bio.terra.service.common.azure;
+
+import bio.terra.common.exception.NotImplementedException;
+import java.security.InvalidParameterException;
+import java.util.UUID;
+
+public enum StorageTableName {
+  SNAPSHOT_TABLE("snapshot"),
+  LOAD_HISTORY_TABLE("loadHistory"),
+  DEPENDENCIES_TABLE("dependencies"),
+  DATASET_TABLE("dataset") {
```

Comment:
I opted to use the static imports and so we could use "SNAPSHOT_TABLE.toTableName()" throughout the code rather than "StorageTableName.SNAPSHOT.toTableName()", but it sounds like static imports are not advised? Why is this the case?

Reply:
I've reworked it to use StorageTableName.SNAPSHOT.toTableName() instead.